### ISSUE TITLE
Directly Using ZDKCreateRequestViewController

### ISSUE THREAD

**Issue by cancan101** [zendesk/zendesk_sdk_ios#141]
Is it possible to directly use  `ZDKCreateRequestViewController`? I am trying:

```objective-c
  ZDKCreateRequestViewController *requestListController = [ZDKCreateRequestViewController new];  
  UINavigationController* navController = [[UINavigationController alloc] initWithRootViewController:requestListController];
  UIViewController *presentingController = ...;
  [presentingController presentViewController:navController animated:YES completion:nil];
```
but then I end up with:
```
2016-01-28 14:35:10.665 qsi[94179:72230317] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSPlaceholderArray initWithObjects:count:]: attempt to insert nil object from objects[0]'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000103810e65 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x000000010265adeb objc_exception_throw + 48
	2   CoreFoundation                      0x00000001036cf652 -[__NSPlaceholderArray initWithObjects:count:] + 290
	3   CoreFoundation                      0x000000010372c7e4 +[NSArray arrayWithObjects:count:] + 52
	4   myapp                                 0x0000000100ff2f98 -[ZDKCreateRequestViewController viewDidLoad] + 292
	5   UIKit                               0x0000000104f0af98 -[UIViewController loadViewIfRequired] + 1198
	6   UIKit                               0x0000000104f0b2e7 -[UIViewController view] + 27
	7   UIKit                               0x0000000104f5d8b0 -[UINavigationController preferredContentSize] + 194
	8   UIKit                               0x0000000104ee20aa -[UIPresentationController preferredContentSizeDidChangeForChildContentContainer:] + 59
	9   UIKit                               0x0000000104ede438 __56-[UIPresentationController runTransitionForCurrentState]_block_invoke + 95
	10  UIKit                               0x0000000104d804a2 _runAfterCACommitDeferredBlocks + 317
	11  UIKit                               0x0000000104d93c01 _cleanUpAfterCAFlushAndRunDeferredBlocks + 95
	12  UIKit                               0x0000000104d9faf3 _afterCACommitHandler + 90
	13  CoreFoundation                      0x000000010373c367 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
	14  CoreFoundation                      0x000000010373c2d7 __CFRunLoopDoObservers + 391
	15  CoreFoundation                      0x0000000103731f2b __CFRunLoopRun + 1147
	16  CoreFoundation                      0x0000000103731828 CFRunLoopRunSpecific + 488
	17  GraphicsServices                    0x0000000107d7fad2 GSEventRunModal + 161
	18  UIKit                               0x0000000104d74610 UIApplicationMain + 171
	19  myapp                                 0x0000000100e9a83f main + 111
	20  libdyld.dylib                       0x000000010652692d start + 1
	21  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

This same approach did work in https://github.com/zendesk/zendesk_sdk_ios/issues/112#issuecomment-155181957 for `ZDKRequestListViewController`.

/CC @RonanMcH @tecknut @baz8080

**Comment by RonanMcH** [comment]
Hi @cancan101 ,
You can use `ZDKCreateRequestViewController` this way but you need to use a different initializer.  If you use: 
```
- (instancetype) initWithSuccess:(ZDKCreateRequestSuccess)success
                        andError:(ZDKCreateRequestError)error
```

To initialize the `ZDKCreateRequestViewController`, it will work.  The callback is called upon a successful or unsuccessful submit of the request.  This is a minor bug that the `init` method for this view controller should be set as `NS_UNAVAILABLE` to point users to the correct initialization code. I will correct this now.
Thanks,
Ronan.

**Comment by schlan** [comment]
Hi @cancan101 , we released `v1.5.4.1` with a fix for your issue in it. I'm going to close this issue, feel free to reopen or creating a new one if you have any further questions.
